setup.py

```diff
@@ -1,10 +1,11 @@
+import distutils
```

Comment:
Is `distutils` a new library that is needed for setup but somehow wasn't before?

Reply:
Sorry, that was a left over from an unneeded change I tried.